### DISCUSSION TITLE
Icon/Symbol alongside text

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2033] [Q&A] [answered]
Is there any way to add a symbol and a text side by side? Something similar to this:

![image](https://github.com/h2oai/wave/assets/117831190/0dfb2333-3a0e-4f2b-8b27-014ac43385b2)

One workaround I can think of is using disabled buttons alongside a `ui.text(`) component. But is there a better way to do this? Thank you!

**Comment by mturoci** [comment]
I would go with `ui.image` (SVG icon) and `ui.text` wrapped within `ui.inline` to put them side by side.

**Reply by n-srinidhi** [reply]
Any way to give colors to the SVG icon? I am getting the black and white default ones,

**Reply by mturoci** [reply, marked as answer]
See the SVG spec, it supports `fill` `stroke` etc. for coloring. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill

**Reply by n-srinidhi** [reply]
Great! Thank you! Will check this out.
